lib/main.coffee

```diff
@@ -93,35 +90,15 @@ module.exports =
 
   # Public: Creates AutocompleteManager instances for all active and future editors (soon, just a single AutocompleteManager)
   activate: ->
-    @getAutocompleteManager()
-    # @activateTimeout = setTimeout(@getAutocompleteManager, 0)
+    run = =>
+      @getAutocompleteManager()
+    setTimeout(run.bind(this), 0)
```

Comment:
Rather than making this async, can we just lighten up the init and run when we need the first completion or something? What's actually the slow part?